books/calculus/bake

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'bundler/inline'
+
+gemfile do
+  gem 'openstax_kitchen', ENV['USE_LOCAL_KITCHEN'] ? { path: '/code/kitchen' } : 'cf0b374a71a1bcc30576988cf07f6b816cb3d0f0'
```

Comment:
I don't think this works to just have the sha at the end, think it would need to look like

```ruby
gem 'openstax_kitchen', ENV['USE_LOCAL_KITCHEN'] ? 
                          { path: '/code/kitchen' } : 
                          (github: 'openstax/kitchen', ref: 'cf0b374a71a1bcc30576988cf07f6b816cb3d0f0')
```

or something -- you should definitely test it.  See https://bundler.io/guides/git.html